src/main/java/com/capitalone/dashboard/service/TestResultServiceImpl.java

```diff
@@ -604,7 +604,8 @@ private String getConfigurationItem(String configurationItem, String targetAppNa
             return configurationItem;
         }
         else if (StringUtils.isNotBlank(targetAppName) ){
-           List<Cmdb> cmdb = cmdbService.commonNameByConfigurationItem(targetAppName);
+            List<Cmdb> cmdb = cmdbService.configurationItemsByTypeWithFilter("", targetAppName,
```

Comment:
Why is the page size set to this large value? it needs to be something manageable

Reply:
page data size is updated to the needed content.